package.json

```diff
@@ -12,8 +12,10 @@
     "@types/react": "^18.0.26",
     "@types/react-router-dom": "^5.1.7",
     "antd": "^5.1.2",
+    "firebase": "^8.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-firebase-hooks": "^5.1.1",
```

Comment:
Please remove `react-firebase-hooks` since we are not using this anymore.

Reply:
did the update on the next pr.